cc_midterm/cc_midterm/templates/base.html

```diff
@@ -12,19 +12,21 @@
         <span><a href="/logout">Logout</a></span>
         &nbsp
 
-        <span style="border-left:1px solid #FFF;height:100%; margin: 6px"></span>
-        &nbsp
-        <span><a href="/engagement-over-time">Engagement Over Time</a></span>
-        &nbsp
+        {% if request.user.is_authenticated %}
```

Comment:
Ah, smart idea! Didn't realize it was that easy haha

Reply:
Hmm.. we should probably add a note that you need to be logged in. I'll make the change quick!